peep_pip.py

```diff
@@ -0,0 +1,117 @@
+import math
+
+from scipy import signal
+
+from ringbuffer import RingBuffer
+
+
+class PEEP():
+    """A filter that takes a pressure waveform and gives the
+    Positive-End Expiratory Pressure (PEEP).
+    """
+
+    def __init__(self,
+                 sampling_period,
+                 maximum_pressure=8.23,
```

Comment:
`8.23` seems like a magic number.  Why this one?

Reply:
And more to the point, shouldn't it go into a constants file?

Reply:
8.23 lined up well with the data that Patrick made for me.  But yeah constants file seems like a reasonable place for it.